src/App.tsx

```diff
@@ -204,10 +204,11 @@ const App = () => {
             style={loading ? styles.containerLoading : styles.container}
             allowsInlineMediaPlayback={true}
             mediaPlaybackRequiresUserAction={false}
-            pullToRefreshEnabled
+            pullToRefreshEnabled={false}
             decelerationRate={0.998}
             startInLoadingState
             javaScriptEnabled
+            bounces={false}
```

Comment:
Likewise, I don't know why this was ever turned on, but holy moly the app feels infinitely better without it!

Great catch ser.